Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM ubuntu:22.04 as build
+FROM ubuntu:24.04 AS build
+
 ENV TZ=UTC
+ENV LANG=C.UTF-8
 
 # Install Required Dependencies
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && \
-    apt-get update && apt-get install -y \
+    # Add source for practicerom-dev install
+    curl https://practicerom.com/public/packages/debian/pgp.pub | apt-key add - && \ 
+    echo deb http://practicerom.com/public/packages/debian staging main >/etc/apt/sources.list.d/practicerom.list &&  \ 
+    apt-get update && \
+    apt-get install -y \
```

Comment:
Not sure on the dependencies of practicerom stuff, but update and install should be at the top.

Reply:
Absolutely agree on the importance of updating ASAP. The lines above this add to the 'list' of things for apt-update to update, so we can have one apt-get update step instead of two as it is currently.